libnixf/src/Sema/VariableLookup.cpp

```diff
@@ -105,6 +134,30 @@ void VariableLookupAnalysis::lookupVar(const ExprVar &Var,
     for (const auto *WithEnv : WithEnvs) {
       Def = WithDefs.at(WithEnv->syntax());
       Def->usedBy(Var);
+
+      // Attempt to resolve the variable with nixpkgs client.
+      bool IsKnown = false;
+      const auto &With = static_cast<const ExprWith &>(*WithEnv->syntax());
+
+      auto FullPath = With.fullPath();
```

Comment:
I construct path string here, then split it. Maybe I should return a list of attr path elements instead and avoid messing with full path string.